eureka-client-jersey3/src/main/java/com/netflix/discovery/shared/transport/jersey3/EurekaJersey3ClientImpl.java

```diff
@@ -232,7 +232,7 @@ class MyDefaultApacheHttpClient4Config extends ClientConfig {
 
                 if (systemSSL) {
                     cm = createSystemSslCM();
-                } else if (trustStoreFileName != null) {
+                } else if (trustStoreFileName != null && !trustStoreFileName.trim().isEmpty()) {
```

Comment:
Unsure why this was needed.

Reply:
Not sure exactly. But it seems to be preventing the file name to be empty string which probably do any harm.